testing/tests/cow_str_implements_fast_writable.rs

```diff
@@ -0,0 +1,33 @@
+use std::borrow::Cow;
+
+use askama::{FastWritable, Template};
+
+// This tests, that Cow<'_, str> implements FastWritable.
+// If it didn't, compiling the tests would fail.
+//
+// Since this test fails at compile-time, there isn't actually any need to run it.
+#[allow(dead_code)]
```

Comment:
```suggestion
#[test]
```

Reply:
The test works compile time though. I could even just put it in a const block.

Reply:
True, however it also allows us to ensure that it's actually built.

Reply:
Why would the test not be built?
With cargo test --workspace it's built.